system-tests/projects/e2e/cypress/e2e/screenshot_viewport_capture.cy.js

```diff
@@ -26,3 +26,61 @@ it('takes consistent viewport captures', () => {
     Cypress._.times(25, fn)
   })
 })
+
+it('properly blacks out absolute elements within a relative container', () => {
+  cy.visit('cypress/fixtures/screenshot-blackout.html')
+  .get('.centered-container')
+  .screenshot({
+    blackout: ['.blue'],
+    onBeforeScreenshot () {
+      const blackedOutElementCoordinates = Cypress.$(
+        '#__cypress-animation-disabler+div.__cypress-blackout',
```

Comment:
I'm open to better selectors here. This seems to be a bit of a coincidence that our blacked out elements are adjacent to the animation disabler element. Also the test assertions below aren't great, but at least should guarantee that the blacked out element is very close to the thing it's trying to black out.